ravif/Cargo.toml

```diff
@@ -22,7 +22,7 @@ rgb = "0.8.29"
 imgref = "1.9.1"
 loop9 = "0.1.3"
 
-[target.'cfg(target_arch = "wasm32")'.dependencies]
+[target.'cfg(target = "wasm32-unknown-unknown")'.dependencies]
 rav1e = { version = "0.5.0", features = ["wasm"] }
```

Comment:
The `wasm` feature of `rav1e` enables `wasm-bindgen` for `rav1e`. `wasm-bindgen` is meant for WASM only used in the browser, where it is fine to have dedicated clue code just for this use case. I am targeting multiple runtimes, and not just the browser, which is why the `wasm-bindgen` clue code isn't working for me. `wasm-bindgen` cannot be used in this case, because it generates multiple WASM imports and exports that are then required but are meant for the clue code.

This is why I'd propose to only enable the `wasm` feature for the `wasm32-unknown-unknown` target only, which is usually the one used when targeting the browser. The other targets are: `wasm32-wasi` and `wasm32-unknown-emscripten`. `wasm32-wasi` is usually not meant for the browser (but could in theory be used there), and `wasm32-unknown-emscripten` comes with its own `emscripten`-generated clue code.

An alternative solution would be to add a feature to `ravif` to toggle `rav1e/wasm`.